src/test/java/ru/yandex/clickhouse/ClickHouseContainerForTest.java

```diff
@@ -33,7 +34,24 @@
             }
             imageTag = ":" + imageTag;
         }
-        clickhouseContainer = new GenericContainer<>("yandex/clickhouse-server" + imageTag)
+
+        /* on java 8 better to replace with
+        *  clickhouseContainer = new GenericContainer<>(
+        *       new ImageFromDockerfile()
+        *               .withDockerfileFromBuilder(builder ->
+        *                               builder
+        *                                      .from("yandex/clickhouse-server" + imageTag )
+        *                                      .run("apt-get update && apt-get install tzdata")
+        *               ))
+        */
+
+        final String fullImageName = "yandex/clickhouse-server" + imageTag;
+        final String runInstallTZ = "RUN apt-get update && apt-get install tzdata";
+
+        ImageFromDockerfile i = new ImageFromDockerfile();
+        i.withFileFromString("Dockerfile", String.format("FROM %s \n %s", fullImageName, runInstallTZ));
+
+        clickhouseContainer = new GenericContainer<>(i)
```

Comment:
Why not put an executable shellscript(e.g. `install-prerequisities.sh`) under `src/test/resources/initdb.d` folder, and mount it into container like below?
```java
.withClasspathResourceMapping("initdb.d", "/docker-entrypoint-initdb.d", BindMode.READ_ONLY)
```

You may need to update `.gitignore` in order to add shellscript, which is something looks like below:
```bash
#!/bin/bash

set -e

if [ ! -f /etc/localtime  ]; then
        apt-get update && DEBIAN_FRONTEND="noninteractive" apt-get -y install tzdata
fi
```

Reply:
My way allows local docker to cache TZdata in docker image cache. 
Your way will download TZdata from Ubuntu repo in runtime always.

Reply:
Good point. But maybe you should check if `/etc/localtime` exists first? Since `tzdata` will be added into official docker image, we no longer need to run apt-get update/install later.

Reply:
> But maybe you should check if /etc/localtime exists first

It is impossible. It wont be a problem / overhead when it will be added into official docker image.
Eventually we will remove this code (apt-get) because CH server will switch to Alpine.